src/plans/create/steps/other-settings/RootDeviceField.tsx

```diff
@@ -48,30 +46,24 @@ const RootDeviceField: FC = () => {
           </ForkliftTrans>
         </HelpIconPopover>
       }
+      helperText={t(
+        'Provide the storage device or partition that contains the root filesystem. If left blank, the first root device will be used.',
+      )}
     >
       <Controller
         name={fieldId}
         control={control}
         render={({ field }) => (
-          <Select
+          <TextInput
             id={field.name}
             value={field.value}
-            onSelect={(_event, value) => {
+            onChange={(_event, value) => {
               field.onChange(value);
             }}
-            placeholder={t('First root device')}
-          >
-            <SelectList>
-              {diskOptions.map(({ description, key }) => (
-                <SelectOption key={key} value={key} description={description}>
-                  {getRootDiskLabelByKey(String(key))}
-                </SelectOption>
-              ))}
-            </SelectList>
-          </Select>
+          />
```

Comment:
Worth adding validations for the text field before applying, specifically if an invalid value might fail the migration (helperTextInvalid, validated..)